src/models/launch_1/README.md

```diff
@@ -0,0 +1,40 @@
+# Launch Configurable Keyboard (launch_1)
+
+- [External Overview](./external-overview.md)
+- [Usage and Customization](./customization.md)
+
+![Launch Photo](./img/launch.png)
+
+The System76 Launch is a configurable keyboard with the following specifications:
+
+- Chassis
+  - [System76 Open Source milled chassis design](https://github.com/system76/launch/tree/master/chassis)
+  - Detachable lift bar to adjust keyboard angle by 15 degrees
+- Electronics
+  - [System76 Open Source PCB design](https://github.com/system76/launch/tree/master/pcb)
+  - Individually addressable RGB LED backlighting
+  - N-key rollover
+- Sockets and Switches
+  - Kailh MX Hotswap sockets
+  - Kailh Box Jade or Box Royal switches
+- Key Caps
+  - PBT plastic
+  - Dye sublimated legend
+  - XDA profile
+- Layout
+  - Modified ANSI US QWERTY
+- Custom Configurations
+  - Customize layout and lighting in firmware with the [System76 Keyboard Configurator](https://github.com/pop-os/keyboard-configurator)
+  - [Available on Linux, Windows, and macOS](https://system76.com/accessories/launch/download)
+- Integrated Hub
+  - 2x USB 3.2 Gen 2 Type-C (Up to 10 Gbps)
```

Comment:
We don't provide "Up To" for the USB ports on any other products; the port generation covers this information. Is it okay to remove that?

Reply:
Yeah I think that's fine